### DISCUSSION TITLE
messaged when running code

### DISCUSSION THREAD

**Discussion by ISoloLlama** [microsoft/vscode-cpptools#6994] [Q&A] [answered]
(electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.

**Comment by sean-mcmanus** [comment, marked as answer]
I have also seen that message -- I think that's from VS Code (electron) and unrelated to our extension, right?
